client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -72,21 +74,26 @@ const DataFilesCopyModal = React.memo(() => {
         section: 'modal'
       }
     });
-    dispatch({
-      type: 'DATA_FILES_SET_MODAL_PROPS',
-      payload: {
-        operation: 'copy',
-        props: {}
-      }
-    });
   };
 
+  const excludedSystems = systems
+    .filter(s => s.scheme !== 'private')
+    .filter(s => s.scheme !== 'public' && !canMakePublic)
```

Comment:
Let's double check the logic in this filter. For instance if a system has the scheme "abcdef", then whether it shows up in `excludedSystems` will depend on the `canMakePublic` variable, which seems wrong. I think the right expression is this, so if a system is public AND the selection can be made public, we exclude it from the exclusion list (meaning it shows up in the dropdown... how confusing!)
```suggestion
    .filter(s => !(s.scheme === 'public' && canMakePublic))
```